src/Parser/RestorationShaman/Modules/ShamanCore/ShamanAbilityTracker.js

```diff
@@ -44,6 +48,22 @@ class ShamanAbilityTracker extends AbilityTracker {
     }
 
   }
+
+  suggestions(when) {
+    const healingRain = this.getAbility(SPELLS.HEALING_RAIN_CAST.id);
+    const has4PT20 = this.owner.selectedCombatant.hasBuff(SPELLS.RESTORATION_SHAMAN_T20_4SET_BONUS_BUFF.id);
+    const unbuffedHealingRainsPercentage = has4PT20 && ((healingRain.casts - healingRain.withT20Buff) / healingRain.casts);
+    if (has4PT20) {
```

Comment:
yeah, I've moved it into T20 4set module. I had put it in the AbilityTracker module is because of the data it used was calculated here, but since we can use `this.owner.modules` to access other module, it's ok to move to a module with closer meaning.